proxy-server/proxy-server.go

```diff
@@ -56,6 +56,7 @@ func getEnv(key, dflt string) string {
 var configTimeStamp time.Time
 
 func respondWithError(res http.ResponseWriter, status int, message string) {
+	log.Printf("Error %d: %s", status, message)
```

Comment:
It is for extra debug, right? We already have fallback codes logged - 
>>Fallback: no cookie token presented or bad cookie. 
If we are to keep it let's put it in:
 if debug { } ?

Reply:
For the cases where there it's not in the fallback code section.